tests/v2/test_rest_relay_api.nim

```diff
@@ -45,7 +45,7 @@ suite "REST API - Relay":
     restServer.start()
 
     let pubSubTopics = @[
-      PubSubTopicString("pubsub-topic-1"), 
+      PubSubTopicString("pubsub-topic-1"),
```

Comment:
can we include in this PR removing this `PubSubTopicString`? I guess the correct way would be PubsubTopic("pubsub-topic-1)?

Reply:
That's too much work. I think this is out of the scope of this PR. As part of one of the subsequent releases, the REST API will be finished, this will be the moment when this code will be cleaned up.

But feel free to do it yourself if you have the bandwidth.

----

That means refactoring the following types:

https://github.com/status-im/nwaku/blob/7ef4eaf46d56bbd08c5a72baed9fdd6d2d3784aa/waku/v2/node/rest/relay/api_types.nim#L20-L22

And all the associated code.